tests/e2e/specs/upload_tapestation_file.js

```diff
@@ -14,5 +14,15 @@ describe('tapestation', () => {
     cy.visit('#/plates')
     cy.contains('compactRegionTable').click()
     cy.get('.row > h3').contains('compactRegionTable')
+
```

Comment:
Rather than having an empty fixture you can just have `{ statusCode: 201, body: {}}`